pyiron_base/jobs/job/extension/executable.py

```diff
@@ -289,6 +283,10 @@ def _executable_versions_list(self):
         Returns:
             dict: list of the available version
         """
+        from pyiron_snippets.resources import ResourceResolver
+        return dict(ResourceResolver(
+            self.path_bin,
+
```

Comment:
This part seems to break the unit tests.

Reply:
Ah, upps. That wasn't supposed to be there yet.